README.md

```diff
@@ -18,128 +18,75 @@ your part.
 -------------------------------------------------------------------------------
 [your text here]
 
 -------------------------------------------------------------------------------
-What's the justification that this really does need to be signed for the whole world to be able to boot it:
+### What's the justification that this really does need to be signed for the whole world to be able to boot it?
 -------------------------------------------------------------------------------
 [your text here]
 
 -------------------------------------------------------------------------------
-Who is the primary contact for security updates, etc.
+### Who is the primary contact for security updates, etc.?
 -------------------------------------------------------------------------------
 - Name:
 - Position:
 - Email address:
 - PGP key, signed by the other security contacts, and preferably also with signatures that are reasonably well known in the Linux community:
 
 -------------------------------------------------------------------------------
-Who is the secondary contact for security updates, etc.
+### Who is the secondary contact for security updates, etc.?
 -------------------------------------------------------------------------------
 - Name:
 - Position:
 - Email address:
 - PGP key, signed by the other security contacts, and preferably also with signatures that are reasonably well known in the Linux community:
 
 -------------------------------------------------------------------------------
-Please create your shim binaries starting with the 15.4 shim release tar file:
-https://github.com/rhboot/shim/releases/download/15.4/shim-15.4.tar.bz2
+### Were these binaries created from the 15.4 shim release tar?
+Please create your shim binaries starting with the 15.4 shim release tar file: https://github.com/rhboot/shim/releases/download/15.4/shim-15.4.tar.bz2
 
-This matches https://github.com/rhboot/shim/releases/tag/15.4 and contains
-the appropriate gnu-efi source.
--------------------------------------------------------------------------------
-[Please confirm]
-
--------------------------------------------------------------------------------
-URL for a repo that contains the exact code which was built to get this binary:
--------------------------------------------------------------------------------
-[your url here]
-
--------------------------------------------------------------------------------
-What patches are being applied and why:
--------------------------------------------------------------------------------
-[your text here]
-
--------------------------------------------------------------------------------
-If bootloader, shim loading is, GRUB2: is CVE-2020-14372, CVE-2020-25632,
- CVE-2020-25647, CVE-2020-27749, CVE-2020-27779, CVE-2021-20225, CVE-2021-20233,
- CVE-2020-10713, CVE-2020-14308, CVE-2020-14309, CVE-2020-14310, CVE-2020-14311,
- CVE-2020-15705, and if you are shipping the shim_lock module CVE-2021-3418
```

Comment:
This, combined with the next two questions, seems like a multi-part "If you're loading grub2". This is now moved to the issue_template [here](https://github.com/tedbrandston/shim-review/pull/1/files#diff-e5aa817955e06f89b32731f49a1ccf2bcd70eb907e023ca0766899cf64d7f9aaR46-R70)